components/automate-cli/pkg/diagnostics/context.go

```diff
@@ -340,9 +343,22 @@ func (c *testContext) adminToken() (string, error) {
 
 		if err == nil {
 			c.Globals.CachedToken = resp.ApiToken
+			c.Globals.TokenID = resp.TokenId
 			break
 		}
 	}
 
 	return c.Globals.CachedToken, nil
 }
+
+func (c *testContext) CleanupAdminToken() error {
+	_, err := c.DoLBRequest(
+		fmt.Sprintf("/apis/iam/v2/tokens/%s", c.Globals.TokenID),
+		lbrequest.WithMethod("DELETE"),
+	)
+	if err != nil {
```

Comment:
Will this catch a non-200 response?

Reply:
Not sure 🤔 I'll double check

Reply:
Added some extra handling for non 200/403/500 responses (DoLBRequest only catches 403 and 500s 💡)